README.md

```diff
@@ -1,12 +1,19 @@
 # Intake: A general interface for loading data
+[![Build Status](https://travis-ci.org/ContinuumIO/intake.svg?branch=master)](https://travis-ci.org/ContinuumIO/intake)  [![Documentation Status](https://readthedocs.org/projects/intake/badge/?version=latest)](http://intake.readthedocs.io/en/latest/?badge=latest)
 
-[![Build Status](https://travis-ci.org/ContinuumIO/intake.svg?branch=master)](https://travis-ci.org/ContinuumIO/intake)
-[![Documentation Status](https://readthedocs.org/projects/intake/badge/?version=latest)](http://intake.readthedocs.io/en/latest/?badge=latest)
+Intake is a lightweight set of tools for loading and sharing data in data science projects.  
+Intake helps you:
+
+* Load data from a variety of formats (see the [current list of known plugins](http://intake.readthedocs.io/en/latest/plugin-directory.html)) into containers you already know, like Pandas dataframes, Python lists, NumPy arrays, and more.
+* Convert boilerplate data loading code into reusable Intake plugins
+* Describe data sets in catalog files for easy reuse and sharing between projects and with others.
+* Share catalog information (and data sets) over the network with the Intake server
+
+Documentation is available at [Read the Docs](http://intake.readthedocs.io/en/latest).
 
 Development Environment
 ----------------------------
  * Create development Python environment.
- * Install [snappy](https://github.com/andrix/python-snappy)
  * `pip install -r requirements.txt`
```

Comment:
Should we also provide an environment.yaml, or a link to the conda recipe?

Reply:
yeah, I would not mind setting up an environment.yaml, but we can do that as part of making some more complete developer docs.